android/assets/jsons/translations/Brazilian_Portuguese.properties

```diff
@@ -7594,65 +7594,68 @@ May buy [buildingFilter] buildings with [stat] for [amount] times their normal P
  # Requires translation!
 [stat] cost of purchasing [buildingFilter] buildings [relativeAmount]% = 
  # Requires translation!
-Production to science conversion in cities increased by 33% = 
+Production to science conversion in cities increased by 33% = Conversão de produção para ciência em cidades aumentado em 33%
  # Requires translation!
-Triggers victory = 
+Triggers victory = Aciona vitória
  # Requires translation!
 Starts with [policy] adopted = 
  # Requires translation!
-[amount] Unit Supply = 
+[amount] Unit Supply = [amount] suprimentos de unidade
  # Requires translation!
 [amount] Unit Supply per [amount2] population [cityFilter] = 
  # Requires translation!
-[amount] Unit Supply per city = 
+[amount] Unit Supply per city = [amount] de suprimentos de unidade por cidade
  # Requires translation!
 Rebel units may spawn = 
  # Requires translation!
 Can be purchased for [amount] [stat] [cityFilter] = 
  # Requires translation!
-Requires a [buildingName] in at least [amount] cities = 
+Requires a [buildingName] in at least [amount] cities = Requer um(a) [buildingName] em pelo menos [amount] cidades
  # Requires translation!
 Must not be next to [terrainFilter] = 
  # Requires translation!
 Creates a [improvementName] improvement on a specific tile = 
  # Requires translation!
 No defensive terrain penalty = 
  # Requires translation!
-May attack when embarked = 
+May attack when embarked = Pode atacar quando embarcado
  # Requires translation!
-No Sight = 
+No Sight =
```

Comment:
Don't remove trailing spaces from lines you don't translate... But, it might even be harmless here. Not sure.

Reply:
Ok, i will solve this.